### ISSUE TITLE
Improvements to Military Units Overview window

### ISSUE THREAD

**Issue by knwpsk** [mainIine/foe-helfer-extension#2272]
Please allow me to suggest some improvements:

1. This window can't be resized. Please allow it. I can't see all of the units I need to see (on third tab), and have to scroll instead - which wastes valuable time when fighting.
2. When I'm fighting in GBG and have this window open, I always want it to show the THIRD tab (count of my units). But **every** time I fight, it flips over to the first tab. Please make it stop doing this, and stay on the tab I chose.
3. On the third tab, it would be great if I can see the units I want to see, and not the others. Let me hide or unhide units such as Brave Warrior or Military Drummer that I never use. 
4. The 4th tab only shows me production of Alcatraz. Can it also show other buildings that provide military units, such as the Royal Bathhouse?
5. What would really be great on the first tab (the one that shows my currently-selected units) is if it also showed how many full-strength units of that type I have in reserve, in a column next to Status, Attack, Defence.

THANK YOU for such an incredibly awesome, can't-do-without-it tool! And thanks for the chance to offer suggestions.

Playing FOE on desktop, Chrome browser.

**Comment by knwpsk** [comment]
Great improvements on the armies tab! Thank you!

**Comment by outoftheline** [comment]
i dont think this is still important, because nowadays you can get so many units and the game has improved over the years (stacked army units etc)
